pkgs/applications/virtualization/nvidia-docker/config.toml

```diff
@@ -0,0 +1,13 @@
+disable-require = false
+#swarm-resource = "DOCKER_RESOURCE_GPU"
+
+[nvidia-container-cli]
+#root = "/run/nvidia/driver"
+#path = "/usr/bin/nvidia-container-cli"
+environment = []
+#debug = "/var/log/nvidia-container-runtime-hook.log"
+ldcache = "/tmp/ld.so.cache"
```

Comment:
Is this directory secure? Since it is world-writeable an attacker might put a different file/symlink here. An alternative could be `/run/docker`

Reply:
I've changed it, in case it gets other uses in the future. Currently it had no real effect though, since the ldconfig resolving (of a mostly fixed list of nvidia-libs) is patched out and done instead via string construction of libname + loaded host-kernel nvidia module version.